_posts/news/2019-10-14-2019-Lodge-Leadership-Development.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: 2019 Lodge Leadership Development
+author: Eoghan Pleis
+date: 2019-10-14
+banner:
+thumbnail: sample.png
+excerpt: Develop your leadership skills while taking a leading role in developing the Lodge's future.
+---
+
+## December 6-8, 2019
+
+This Lodge Leadership Development (LLD) will not be one to miss! On the heals of [National OA's committment to Thrive](https://oa-bsa.org/article/thrive-national-webinar-local-commitment), we will be tackling the ambitious answer to "What's Next?" Many hands make light work, and getting many voices in on this discussion will surely help move the conversation forward.
+
+Many of the conversations and trainings will help participants learn how to better support the Lodge, Scouting, and many other communities they may find themselves a part of.
+
+More information will be coming shortly for registration, so stay tuned!
```

Comment:
Do we at least have a general location?

Reply:
We do. I wasn't trying to make the location bit a standard article thing but I guess it makes sense when the article is explicitly about an event.

Reply:
This as fixed now as well